### ISSUE TITLE
How to pass root CA certificate using AWS IOT SDK for Java

### ISSUE THREAD

**Issue by Amit0903** [aws/aws-iot-device-sdk-java#71]
Hi,
I want to connect to AWS IOT using the root CA certificate, device certificate and device private key. However I am not able to find any function in the Java SDK which will create a MQTT Client using these. Even the Samples do not have such example. Its only asked to pass the device certificate and private key. Can you please let me know how to pass the root CA certificate which is the AWS IoT root certificate?

**Comment by mkonnov** [comment]
@Amit0903 
Hello.
AWS is not returning CA certificate, but you can easily fetch it from https://www.symantec.com/content/en/us/enterprise/verisign/roots/VeriSign-Class%203-Public-Primary-Certification-Authority-G5.pem

**Comment by Amit0903** [comment]
Thank you.
I also came to know that we don't need to pass the root CA cert specifically as it is present in the java keystore. I was also able to connect to IOT from my mobile app by creating and placing the keystore containing the device private key and certificate on my mobile file system.
I referred the SDK samples at the following path: https://github.com/awslabs/aws-sdk-android-samples/tree/master/AndroidPubSub
